nautobot/core/settings.json

```
@@ -283,6 +316,69 @@
             "type": "boolean",
             "default": false
         },
+        "LOGGING": {
+            "type": "object",
+            "properties": {
+                "version": {
+                    "type": "integer",
+                    "default": 1
+                },
+                "disable_existing_loggers": {
+                    "type": "boolean",
+                    "default": false
+                },
+                "formatters": {
+                    "type": "object",
+                    "default": {
+                        "normal": {
+                            "format": "%(asctime)s.%(msecs)03d %(levelname)-7s %(name)s :\n  %(message)s",
+                            "datefmt": "%H:%M:%S"
+                        },
+                        "verbose": {
+                            "format": "%(asctime)s.%(msecs)03d %(levelname)-7s %(name)-20s %(filename)-15s %(funcName)30s() :\n  %(message)s",
+                            "datefmt": "%H:%M:%S"
+                        }
+                    },
+                    "additioinalProperties": true
```

```suggestion
                    "additionalProperties": true
```

Same throughout.

It would be good for a test to also load this schema into `jsonschema` and ensure that it's a valid schema - a la https://github.com/nautobot/nautobot/blob/develop/nautobot/extras/models/models.py#L276-L279.